### PR TITLE
[Done] Translation of data-fetching.md ..

### PR DESCRIPTION
**Pull request by 3imed-jaberi**
If there is a better expression than ` التطبيقات المبنية على البيانات` to translate the `data-driven app` .. just suggest  ...

### CODE REVIEW

File: content/community/tools-data-fetching.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 ---
 
-* **[Apollo](https://www.apollographql.com/docs/react/):** Easy to set up and use GraphQL client.
-* **[Axios](https://github.com/mzabriskie/axios):** Promise based HTTP client for the browser and node.js.
-* **[Relay Modern](https://facebook.github.io/relay/docs/en/new-in-relay-modern.html)** - A JavaScript framework for building data-driven React applications.
-* **[Request](https://github.com/request/request):** Simplified HTTP request client.
-* **[Superagent](https://visionmedia.github.io/superagent/):** A lightweight "isomorphic" library for AJAX requests.
+* **[Apollo](https://www.apollographql.com/docs/react/):** سهل الإعداد واستخدام عميل GraphQL.
+* **[Axios](https://github.com/mzabriskie/axios):** عميل HTTP يستند إلى الوعد ( Promise ) للمتصفح و نود جي إس.
+* **[Relay](https://facebook.github.io/relay/docs/en/new-in-relay-modern.html)** - إطار جافا سكريبت لإنشاء تطبيقات React المبنية على البيانات.
```
Comment (Fcmam5) [inline content/community/tools-data-fetching.md]: Framework is `إطار عمل` not just `frame/إطار`

```suggestion
* **[Relay](https://facebook.github.io/relay/docs/en/new-in-relay-modern.html)** - إطار عمل جافا سكريبت لإنشاء تطبيقات React المبنية على البيانات.
```

File: content/community/tools-data-fetching.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 permalink: community/data-fetching.html
 ---
 
-* **[Apollo](https://www.apollographql.com/docs/react/):** Easy to set up and use GraphQL client.
-* **[Axios](https://github.com/mzabriskie/axios):** Promise based HTTP client for the browser and node.js.
-* **[Relay Modern](https://facebook.github.io/relay/docs/en/new-in-relay-modern.html)** - A JavaScript framework for building data-driven React applications.
-* **[Request](https://github.com/request/request):** Simplified HTTP request client.
-* **[Superagent](https://visionmedia.github.io/superagent/):** A lightweight "isomorphic" library for AJAX requests.
+* **[Apollo](https://www.apollographql.com/docs/react/):** سهل الإعداد واستخدام عميل GraphQL.
+* **[Axios](https://github.com/mzabriskie/axios):** عميل HTTP يستند إلى الوعد ( Promise ) للمتصفح و نود جي إس.
```
Comment (Fcmam5) [inline content/community/tools-data-fetching.md]: Just for SEO
```suggestion
* **[Axios](https://github.com/mzabriskie/axios):** عميل HTTP يستند إلى الوعد ( Promise ) للمتصفح و نود جي إس (Node.js).
```

File: content/community/tools-data-fetching.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
 ---
 id: data-fetching
-title: Data Fetching
+title: جلب البيانات
```
Comment (Fcmam5) [inline content/community/tools-data-fetching.md]: This translation is the best translation for 'fetch' I've ever read :) we should add it to the Glossary and agree on using it

File: content/community/tools-data-fetching.md (hunk 1 above)
Comment (3imed-jaberi) [reply content/community/tools-data-fetching.md]: it's a typo 😁 ..

**Pull request merged**